### PR TITLE
feat: use correct password reset title in email

### PR DESCRIPTION
**Pull request by danielgent**
I'm not sure if this fixes the _"issue"_ that Markus may or may not be experiencing, but it has bothered me we say _welcome to your account_ on a reset password email :saluting_face:

### CODE REVIEW

File: src/email-templates/password-reset.mjml [hunk 1]
```diff
@@ -63,7 +63,7 @@
     <mj-section padding="4px">
       <mj-column padding="16px 0px" border-radius="16px" width="504px" background-color="#fff">
         <mj-text font-size="18px" line-height="21.6px" font-weight="500" color="#14141A">
-          {{welcomeToYourAccount}}
+          {{passwordResetTitle}}
```
Comment (danielgent) [inline src/email-templates/password-reset.mjml:66]: Seems more sensible! This is what we're using for the email subject

File: src/templates/email/password-reset.liquid
Comment (danielgent) [inline src/templates/email/password-reset.liquid:1]: We don't actually need these liquid files anymore...

File: test/integration/flows/__image_snapshots__/password-spec-ts-test-integration-flows-password-spec-ts-password-flow-password-reset-should-send-password-reset-email-for-existing-user-and-allow-password-to-be-changed-1-snap.png
Comment (danielgent) [inline test/integration/flows/__image_snapshots__/password-spec-ts-test-integration-flows-password-spec-ts-password-flow-password-reset-should-send-password-reset-email-for-existing-user-and-allow-password-to-be-changed-1-snap.png:1]: voila.

**Review by markusahlstrand — APPROVED** [review]
LGTM

**Pull request merged**